src/composables/useFlawAffectsModel.ts

```diff
@@ -12,12 +12,17 @@ import { useToastStore } from '@/stores/ToastStore';
 import type { ZodFlawType } from '@/types/zodFlaw';
 import type { ZodAffectType, ZodAffectCVSSType } from '@/types/zodAffect';
 import { deepCopyFromRaw } from '@/utils/helpers';
+import { sortWith, ascend, prop } from 'ramda';
+
+const sortAffects = (affects = []) => {
+  return sortWith([ascend(prop('ps_module')), ascend(prop('ps_component'))], affects);
```

Comment:
I think this is not entirely the thing, which is asked for. To sort by product family, you need to use `Affect.ps_product` field which was added to OSIDB API and for example things like rhel-8 and rhel-9 will both fall under Red Hat Enterprise Linux product family. Will send a SFM2 reference in slack.

Reply:
Thanks @JakubFrejlach  I have one question. what happen if ps_product is null, are we showing those affects at the top or bottom in SFM2?